### DISCUSSION TITLE
Universal Render Pipeline 사용 여부

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#15] [5. Q&A]
안녕하세요,
제페토 패키지 내에서 Universal Render Pipeline를 사용할 수 있나요?

**Comment by pbatlhazer** [comment]
Standard Shader를 사용하고 있고, urp나 hdrp는 지원하고 있지 않는다고 합니다.
https://docs.unity3d.com/kr/530/Manual/shader-StandardShader.html

제페토에서 지원하는 쉐이더는 아래의 가이드를 참고하시면 될것 같아요.
https://studio.zepeto.me/guides/creating-your-item-custom-shader
